### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Maksym845**
- [DEMO LINK](https://Maksym845.github.io/react_tabs/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -10,33 +11,21 @@ export const tabs = [
...
+  const [selected, setSelected] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {tabs.filter(tab => tab.id === selected.id)
+          .map(tab => tab.title)}
```
Comment (Yeliseiev) [inline src/App.tsx:22]: Why you need this filter? You could just use selected.title

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1 +1,57 @@
...
+                </a>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.filter((tab: Tab) => tab.id === selectedTabId)
+          .map((tab: Tab) => tab.content)}
```
Comment (Yeliseiev) [inline src/components/Tabs/Tabs.tsx:53]: You can use find in there.

**Review by Yeliseiev — APPROVED** [review]
Good job!

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -1 +1,57 @@
...
+          {tabs.map((tab: Tab, index: number) => {
+            const { id, title } = tab;
+            const idIsEqual = selectedTabId === id;
+
+            return (
+              <li
+                className={classNames(
+                  {
+                    'is-active': idIsEqual
+                      || (!isCurrentTab && !index),
```
Comment (sergii-nosachenko) [inline src/components/Tabs/Tabs.tsx:32]: Overloaded condition. Here you can simply check if `idIsEqual`.

File: src/components/Tabs/Tabs.tsx (hunk 2 above)
Comment (sergii-nosachenko) [inline src/components/Tabs/Tabs.tsx:53]: Unnecessary filter usage. If you will have `currentTab` variable you can simply use it here.

**Pull request closed without merging**